elife/scripts/root-upgrade-postgresql-9.4-to-10.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
```

Comment:
have never done this, so can't provide much feedback unfortunately

Reply:
I found some evidence I may have done it with psql 8 to 9, but I don't remember having done it either to be honest.